cub/test/catch2_test_device_merge.cu

```diff
@@ -172,62 +77,79 @@ struct fallback_test_policy_hub
 };
 
 // TODO(bgruber): This test alone increases compile time from 1m16s to 8m43s. What's going on?
-C2H_TEST("DeviceMerge::MergeKeys large key types", "[merge][device]", c2h::type_list<large_type_vsmem, large_type_fallb>)
+// C2H_TEST("DeviceMerge::MergeKeys large key types", "[merge][device]", c2h::type_list<large_type_vsmem,
+// large_type_fallb>)
+// {
```

Comment:
I assume this is disabled for CI testing. Please re-enable the test before merging.

Reply:
Thanks for the reminder! I'll make sure to re-enable before merging.